casper/protocols/integer/integer_plot_tool.py

```diff
@@ -71,15 +71,11 @@ def _update_message_labels(self, new_messages):
             self.message_labels[message] = message.estimate
 
     def _update_message_fault_tolerance(self):
-        for validator in self.view.latest_messages:
-
-            latest_message = self.view.latest_messages[validator]
-
-            if latest_message in self.bet_fault_tolerance:
-                continue
-
-            oracle = CliqueOracle(latest_message, validator.view, self.validator_set)
-            fault_tolerance, num_node_ft = oracle.check_estimate_safety()
-
-            if fault_tolerance > 0:
-                self.bet_fault_tolerance[latest_message] = num_node_ft
+        if self.view:
```

Comment:
When does not have a `view`?

Reply:
Good catch. This was an issue in a previous version of the architecture, but I fixed it. Removing!